moving server options to references (#1549)
temporalio/documentation: docs-src/references/server-options.md

---
id: server-options
title: Temporal Server options
sidebar_label: Start options
---

> You can run the Temporal server as a Go application by including the server package `go.temporal.io/server/temporal` and using it to create and start a new Temporal Server.

Reviewer: ```suggestion
You can run the Temporal server as a Go application by including the server package `go.temporal.io/server/temporal` and using it to create and start a Temporal Server.
```
"create...a ~~new~~" per https://developers.google.com/style/word-list#create-new

> Note, there are many ways to [run a Temporal Cluster](/kb/install-temporal-cluster-for-development) and this is approach is recommended for a limited number of situations.

Reviewer: ```suggestion
You can [run a Temporal Cluster](/kb/install-temporal-cluster-for-development) in various ways. We recommend this approach for a limited number of situations.
```
